client/src/components/gamedashboard/top/MarsLog.vue

```diff
@@ -36,7 +36,9 @@ export default class MarsLog extends Vue {
   marsLogColor(log: MarsLogData) {
     console.log(log.category)
     switch (log.category) {
-      case 'upkeep':
+      case 'System Health: Gain':
```

Comment:
instead of explicit `System Health: Gain` and `System Health: Drop` categories can we base this logic on whether the payload integer value is positive or negative?

Reply:
disregard this comment, I realized you're doing this off the logs which don't have that content.